helm-chart/templates/spire-agent.yaml

```diff
@@ -76,10 +78,21 @@ spec:
           while $(curl -t 'DUMMY=1' -s telnet://spire-server:8081); [ $? -ne 48 ]; do
             sleep 1
           done
+      - name: chown-volume
```

Comment:
I'm curious about this volume change. All of our sidecars also use this volume mount, so do we now have to change permissions on our sidecars? Are they too permissive?

Reply:
Hm I see. That's a bummer.